aws/logs_monitoring/enhanced_lambda_metrics.py

```diff
@@ -357,7 +357,7 @@ def acquire_s3_cache_lock():
         if last_modified_unix_time + DD_S3_CACHE_LOCK_TTL_SECONDS >= time():
             return False
     except Exception:
-        logger.exception("Unable to get cache lock file")
+        logger.debug("Unable to get cache lock file")
```

Comment:
Can you update other places where handled exceptions are logged as exception as well? Generally speaking, we should try to not logging anything, unless log level is set to debug, because we can easily generate tons of repetitive logs and inflate customers' CW bills.

Reply:
I think you need to manually include the traceback now that you switched it to debug, perhaps by setting `exc_info=True`.